README.md

```diff
@@ -43,7 +43,16 @@ This creates:
 * a Subcription to the OCS catalog in the `openshift-storage`
 namespace
 
-Once this is done, a StorageCluster can be created.
+You can check the status of CSV by command `oc get csv -n openshift-storage`
+The output should look like:
+
+```console
+NAME                  DISPLAY                                VERSION   REPLACES   PHASE
+ocs-operator.v0.0.1   Openshift Container Storage Operator   0.0.1                Succeeded
+```
```

Comment:
A `console` type code-block is used to show shell contents. So please include the command in it as well, like below.
```console
$ oc get csv -n openshift-storage
NAME                  DISPLAY                                VERSION   REPLACES   PHASE
ocs-operator.v0.0.1   Openshift Container Storage Operator   0.0.1                Succeeded
```

Reply:
done